.travis.yml

```diff
@@ -12,4 +12,6 @@ cache:
     - '.cache'
 
 script:
-  - yarn run test
\ No newline at end of file
+  - yarn run test
+  - yarn run format-check
```

Comment:
could we move the format-check + tslint to be run by test in the package.json?

Reply:
In my opinion we should keep the test script in package.json as it is. This allows us to pass options to it during development, like `--update-snapshot` or `--coverage`. I added a sperate `test-ci` script that runs all of these scripts instead 🙂

Reply:
I think it is important that folks running the tests locally have the linter run.

perhaps we could split the difference and make a jest script... that way if you want to pass args you can do so from that script?

or alternatively have jest as the last command, which is where arguments will be passed if you do `yarn test -- arg1 arg2 arg3`

Reply:
That makes a lot of sense. I'll update this in a minute 👍